components/camel-consul/src/test/java/org/apache/camel/component/consul/cluster/ConsulClusteredRoutePolicyFactoryIT.java

```diff
@@ -94,7 +95,7 @@ public void configure() {
 
             // Start the context after some random time so the startup order
             // changes for each test.
-            Thread.sleep(ThreadLocalRandom.current().nextInt(500));
+            Awaitility.await().atMost(ThreadLocalRandom.current().nextInt(500), TimeUnit.MILLISECONDS);
```

Comment:
This is missing the terminal operation. You need to add something like `untilAsserted`, `until` or similar.

I believe we need something similar to this:


```
Awaitility.await().pollDelay(ThreadLocalRandom.current().nextInt(500), TimeUnit.MILLISECONDS).untilAsserted(() -> Assertions.assertDoesNotThrow(context::start));
```